spec/features/guidance_after_sign_in_spec.rb

```diff
@@ -16,7 +17,7 @@
 
   it 'displays radius connection details' do
     expect(page).to have_content(
-      "Your RADIUS secret is #{user.organisation.locations.first.radius_secret_key}"
+      user.organisation.locations.first.radius_secret_key
```

Comment:
This could probably just be:
location.radius_secret_key ?

Reply:
👍